internal/output/output_result.go

```diff
@@ -401,10 +402,16 @@ func processPackage(vulnPkg models.PackageVulns) PackageResult {
 
 	packageFixedVersion := calculatePackageFixedVersion(vulnPkg.Package.Ecosystem, regularVulnList)
 
+	installVersion := vulnPkg.Package.Version
+	// If the ecosystem is not set, it's most likely from Git
+	if installVersion == "" && vulnPkg.Package.Ecosystem == "" {
+		installVersion = results.GetShortCommit(vulnPkg.Package.Commit)
```

Comment:
We should save the full commit here. This way in the HTML report we can have this as a hover tooltip.

Reply:
And if there is no package name/repo name, we should be showing the full commit.

Reply:
updated code to show short commit with the full commit in a hover tooltip. I think as long as the full commit is in the tooltip, it doesn't matter if there is a package name or not.